system_strings/sysmes_text.en

```diff
@@ -1284,14 +1284,14 @@ I will never do anything that^will make Arcueid cry ever again.^Even if my brain
 ロアの死体が焼かれていく。^街を脅かす脅威は去った。^吸血鬼事件は終わったのだ。
 これで終わり。^明日からは平穏に生きられる。^傷や思い出も、全部忘れて――
 クリスマスのプレゼント。^代行者としての責務と、^復讐者としての情熱と。
-'It's best if we don't see each^other anymore.' Arcueid said^that. As a human, that would^definitely be the right choice.
+Arcueid said 'It's best if we don't see each^other anymore.'^As a human, that would^definitely be the right choice.
 It's after-school. The rest of^the day is mine. Then, what I^should do now is――
-I should pretend none of this^ever happened. I should think^of all of this as a dream. Is^that really―――really it?
+I should pretend none of this^ever happened like all of this^was a just dream. Is that really―――really it?
```

Comment:
Needs some sort of punctuation after `happened`. Either a period or some em dashes